switchapi/es2k/lnw_v3/switch_pd_lag.c

```diff
@@ -580,6 +581,15 @@ switch_status_t switch_pd_tx_lacp_lag_table_entry(
     goto dealloc_resources;
   }
 
+  status = tdi_data_field_id_with_action_get(
+      table_info_hdl, ACTION_SET_EGRESS_PORT_PARAM_ROUTER_INTF_ID, action_id,
+      &rif_data_field_id);
+  if (status != TDI_SUCCESS) {
+    krnlmon_log_error("Unable to get data field id param for: %s, error: %d",
+                      ACTION_SET_EGRESS_PORT_PARAM_EGRESS_PORT, status);
```

Comment:
ACTION_SET_EGRESS_PORT_PARAM_EGRESS_PORT -> Change this to ACTION_SET_EGRESS_PORT_PARAM_ROUTER_INTF_ID for correct error message.

Reply:
Open PR #156 to address this